pkg/config/config.go

```diff
@@ -4,6 +4,7 @@ package config
 
 import (
 	"github.com/go-logr/logr"
+
```

Comment:
```suggestion
```
No need for a new line. Both are 3rd party imports. We should not be treating the GIE differently than other 3rd party imports. IMO.